File: Data Structures & Algorithms/concatenation-of-array/submission-1.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int newLen = 2 * nums.length;
        int[] ans = new int[newLen];

        for (int i = 0; i < newLen; i++) {
            ans[i] = nums[i % nums.length];
        }

        return ans;
    }
}

File: Data Structures & Algorithms/remove-element/submission-1.py
class Solution:

    def removeElement(self, nums: List[int], val: int) -> int:
        without = sorted([x for x in nums if x != val])

        k = len(without)

        for i in range(len(without)):
            nums[i] = without[i]

        return k
        
            
            
        

